src/domain/valueObject/cronSchedule.go

```diff
@@ -14,6 +16,11 @@ func NewCronSchedule(value string) (CronSchedule, error) {
 	if !schedule.isValid() {
 		return "", errors.New("InvalidCronSchedule")
 	}
+
+	if !schedule.hasAtSign() {
```

Comment:
shouldHaveAtSign seria um nome melhor... você pode inclusive botar isso acima do isValid, daí você pode tornar o @ obrigatório quando for frequência pré-definida.

Reply:
Anotado.

Reply:
Ajustado. Arquivo `outdated`. Aguardando revisão.